vscode-web/main.tf

```diff
@@ -160,7 +169,7 @@ resource "coder_app" "vscode-web" {
   display_name = var.display_name
   url          = var.folder == "" ? "http://localhost:${var.port}" : "http://localhost:${var.port}?folder=${var.folder}"
   icon         = "/icon/code.svg"
-  subdomain    = true
+  subdomain    = var.subdomain
```

Comment:
Setting subdomain to false breaks the app. I think you'll have to chang the URL format when it's set.